x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/ignore_z_value_parameter.tsx

```diff
@@ -10,15 +10,19 @@ import { i18n } from '@kbn/i18n';
 
 import { EditFieldFormRow } from '../fields/edit_field';
 
-export const IgnoreZValueParameter = () => (
+export const IgnoreZValueParameter = ({ description }: { description?: string }) => (
   <EditFieldFormRow
     title={i18n.translate('xpack.idxMgmt.mappingsEditor.ignoreZValueFieldTitle', {
       defaultMessage: 'Ignore Z value',
     })}
-    description={i18n.translate('xpack.idxMgmt.mappingsEditor.ignoredZValueFieldDescription', {
-      defaultMessage:
-        'Three dimension points will be accepted, but only latitude and longitude values will be indexed; the third dimension is ignored.',
-    })}
+    description={
+      description
+        ? description
```

Comment:
I think a `||` operator could be used here or `??` if you only want to show default message for `null` and `undefined` values.